packages/core/src/utils/assertion.ts

```diff
@@ -80,6 +80,8 @@ export const isRootKey = (specKey: string, target: string) => {
   return specKey === target;
 };
 
+const LOCALHOST_REGEX = /^(https?|ftp):\/\/[^\s/$.?#].[^\s]*$/i;
+
 export const isUrl = (str: string) => {
-  return validatorIsUrl(str, { require_tld: false });
+  return validatorIsUrl(str) || LOCALHOST_REGEX.test(str);
```

Comment:
Too bad to not use only the validator. Which case is not handled by it?

Reply:
So with your `require_tld` fix it broke the files so this test fails `expect(isUrl('D:/a/test.txt')).toBeFalsy();` it says that is a URL.

Reply:
My new fix catches your scenario as well as the legacy localhost scenarios and all tests are passing on Windows.

Reply:
That's weird for sure. It should be investigated in the validator lib. I'll try to do it soon.

Reply:
Yeah i went and looked and there a tons of bugs in validator about isUrl and the real issue was that your `http://my-docker-service/docs.json` was not being reported as a URL when it clearly is a URL is because of a bug about dashes in `url`.  If you add `.com` to your URL it works but to me that validator lib has a bunch of lot of bugs.

here is  Regex tester you can use for my Regex: https://regex101.com/r/aB2hcG/1